js/language.js

```diff
@@ -87,10 +87,10 @@ class Localization {
         function deepAssign(target, source) {
             // Object.assign() but deep-assigning objects recursively
             for (let [key, val] of Object.entries(source)) {
-                if (typeof val === "object") {
-                    deepAssign(target[key], val);
-                } else {
+                if (target[key] === undefined || typeof val !== "object") {
```

Comment:
In our case scenario it should never happen that the target doesn't contain a key that exists in the source object.
This would mean that the main (english) localization doesn't have an entry that's present in another localization (`export_props` in dutch here for example).